lib/workers/branch/index.ts

```diff
@@ -148,14 +149,20 @@ export async function processBranch(
     }
     if (
       !branchExists &&
```

Comment:
If branch exists then doesn't apply to PR limits as they are enforced on creation